tests/unit/test_signature.cpp

```diff
@@ -35,27 +36,41 @@ class SignatureTest : public ::testing::Test
     void SetUp() override;
 
 protected:
-    std::unique_ptr<AsymmetricKeypair> _keyPair;
+    std::unique_ptr<AsymmetricKeypair> _keyPairRsa;
```

Comment:
Small nitpick that applies to the whole file: I think RSA and ECC should always be fully capitalized.

Reply:
![](https://media0.giphy.com/media/5hkCGzg4jjQcEXQ6R9/giphy.gif)